test/brief.jl

```diff
@@ -27,22 +27,22 @@ facts("BRIEF") do
     end
 
     context("Descriptor Calculation") do
-        img_1 = Gray{Float64}[ 0 0 0 0 0 0 
-                  0 0 1 1 0 0 
-                  0 0 1 1 0 0 
-                  0 0 1 1 0 0 
-                  0 0 1 1 0 0 
-                  0 0 0 0 0 0 ]
+        #=img_1 = Gray{Float64}[  0 0 0 0 0 0 
```

Comment:
Yeah, that should work. @annimesh2809 you could see the tests which follow this one for more examples. To add variety, we can add some other shape like a triangle since the other tests cover this one.